### ISSUE TITLE
Display how many blocks remain for recovery

### ISSUE THREAD

**Issue by pythcoiner** [wizardsardine/liana#1161]
Not displayed in `Home` screen nor in `Settings/recovery`

![image](https://github.com/wizardsardine/liana/assets/124568858/a026b42c-c66b-473a-a91e-5ea2e920c31c)

 
![image](https://github.com/wizardsardine/liana/assets/124568858/3067579e-bbc6-4a93-9231-315dc86ee1d7)

**Comment by darosior** [comment]
Don't we already have an issue for this?

**Comment by darosior** [comment]
Also this information is already available in the coins' details, i don't think we should show it everywhere.

**Comment by nondiremanuel** [comment]
Maybe we could give the possibility to be redirected directly to the "coins" page for more details (if the user is not aware of what such page is for).

**Comment by kloaec** [comment]
Yes, I tend to agree with a Coin view redirection, because each coin has different lock.

I agree with the fact that the information isn't really helpful currently, the way it's phrased. We should just say "Move your coins to keep the timelock active "refresh"" or something better.
